test/webtest/src/test/java/com/onfido/qa/websdk/test/ProofOfAddressIT.java

```diff
@@ -105,7 +105,7 @@ public void testPoAUIElements(PoADocumentType documentType) {
 
         verifyCopy(intro.requirementsHeader(), "poa_intro.subtitle");
         assertThat(intro.firstRequirement()).isEqualTo("Shows your current address");
-        assertThat(intro.secondRequirement()).isEqualTo("Matches the address you used on signup");
+        assertThat(intro.secondRequirement()).isEqualTo("**Matches** the address you used on sign up");
```

Comment:
This didn't seem to work: [see percy](https://percy.io/a48652af/web-sdk/builds/20730413/changed/1160785438?browser=edge&browser_ids=18%2C22%2C23%2C25&subcategories=unreviewed%2Cchanges_requested&viewLayout=side-by-side&viewMode=new&width=500&widths=500)

Reply:
👍 good catch I never look at percy

Reply:
done